blocks/engine-specifications/performance-chart.js

```diff
@@ -1,90 +1,60 @@
 // STYLING
+const COLORS = {
+  lineHP: '#78B833',
+  lineTQ: '#004FBC',
+  fillHP: '#C8E691',
+  fillTQ: '#76BAFF',
+  background: '#f7f7f7',
+};
 
-const colorLineHP = '#78B833';
-const colorLineTQ = '#004FBC';
-
-const colorFillHP = '#C8E691';
-const colorFillTQ = '#76BAFF';
-
-const colorBackground = '#f7f7f7';
-
-const strokeWidth = 3;
+const STROKE_WIDTH = 3;
 
 // MATH
+const CONVERSION_FACTORS = {
+  HP: 0.815,
+  TQ: 0.22,
+};
 
-// These are to be used as a way to translate the values to chart positions.
-const conversionFactorHP = 0.815;
-const conversionFactorTQ = 0.15;
-// These are used to get 2 points in between each value to make the line curve.
-const bezierFactor1 = 0.3;
-const bezierFactor2 = 0.6;
+const BEZIER_FACTORS = {
+  factor1: 0.3,
+  factor2: 0.6,
+};
 
 // FUNCTIONS
-
 // From the array of values, this extrapolates 4 on each side to use as fading border.
 const createFakeValues = (type, values) => {
```

Comment:
can you change this confusing name? if it is not creating fake values, have that "fake" in the name is confusing for me. could be name like "createChartValues" or something like this